bin/install

```diff
@@ -57,12 +57,19 @@ get_java() {
 	local destdir=$3
 
 	local url=
+	local signature=
+	local sigkey=
 	local base=http://download.oracle.com/otn-pub/java/jdk/
 	local oracle_version=$(echo $version | sed -e 's/\./u/g')
 
 	local s3base=https://s3-eu-west-1.amazonaws.com/repo.proemion.com
 
 	case $version in
+		corretto-8.202.08.2)
+			url=https://d2znqt9b1bc64u.cloudfront.net/amazon-corretto-8.202.08.2-linux-x64.tar.gz 
+			signature=https://d2znqt9b1bc64u.cloudfront.net/amazon-corretto-8.202.08.2-linux-x64.tar.gz.sig
```

Comment:
Just as recommendation, we may want to add a version variable here? Just to make updates easier to keep signatures and downloaded version in sync

Reply:
@SISheogorath done :-)